src/brevitas/core/bit_width/parameter.py

```diff
@@ -88,11 +89,21 @@ def forward(self) -> Tensor:
 
     def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
                               missing_keys, unexpected_keys, error_msgs):
+        bit_width_const_key = prefix + 'bit_width'
         bit_width_offset_key = prefix + 'bit_width_offset'
+        if bit_width_const_key in state_dict:
+            assert bit_width_offset_key not in state_dict, "both should not be true"
+            state_dict = OrderedDict(
+                [(bit_width_offset_key, v - self.bit_width_base) if k == bit_width_const_key
+                                                                 else (k, v)
+                                                                 for k, v in state_dict.items()]
+            )
```

Comment:
For future reference, we avoid dangling ) in the codebase.

Reply:
Noted, thanks!